README.md

```diff
@@ -85,11 +85,11 @@ Live demos can be added to content using the "live-demo.html" include, such as:
 
 There are several options for this include file - please see live-demo.html for details.
 
-The source for each live demo is store in the `_includes/live-demos/` directory.
+The source for each live demo is store in the `modules/ROOT/examples/live-demos/` directory.
 
 #### Overriding the tinymce URL in live demos
 
-All live demos usually get their tinymce.min.js URL from the `tinymce_live_demo_url` setting in the _config.yml file.
+All live demos usually get their `tinymce.min.js` URL from the `tinymce_live_demo_url` setting in the `antora.yml` file.
```

Comment:
The section above this needs fixing as it still uses the old demo syntax and mentions it's a WIP.

- [ ] Fix live demo section above